adapters/outbrain/outbrain.go

```diff
@@ -43,8 +43,10 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 			errs = append(errs, err)
 			continue
 		}
-		imp.TagID = outbrainExt.TagId
-		reqCopy.Imp[i] = imp
+		if outbrainExt.TagId != "" {
```

Comment:
Please add a new json test for this condition, to be sure the tests cover both tagid = "" and tagid = "<something else>"